HIP/hip-765.md

```diff
@@ -6,7 +6,8 @@ working-group: Ashe Oro (@Ashe-Oro), Michiel Mulders (@michielmulders)
 type: Standards Track
 category: Service
 needs-council-approval: Yes
-status: Accepted
+status: Final
+release: v0.49.0
```

Comment:
Should we put the version that actually got deployed to mainnet (`v0.49.7`) or the version that the feature was first introduced in (`v0.49.0`)? Same comment for the other 2 metadata PRs.